common/js/src/nacl-module/nacl-module.js

```diff
@@ -161,101 +157,105 @@ NaclModule.prototype.disposeInternal = function() {
   const mimeType = this.getMimeType_();
   goog.log.fine(
       this.logger_, 'Preparing NaCl embed (MIME type: "' + mimeType + '")...');
   return goog.dom.createDom(
       'embed',
       {'type': mimeType, 'width': 0, 'height': 0, 'src': this.naclModulePath});
-};
+}
 
 /**
  * @return {string}
  * @private
  */
-NaclModule.prototype.getMimeType_ = function() {
+getMimeType_() {
   switch (this.type) {
-    case NaclModule.Type.NACL:
+    case Type.NACL:
       return 'application/x-nacl';
-    case NaclModule.Type.PNACL:
+    case Type.PNACL:
       return 'application/x-pnacl';
     default:
       GSC.Logging.failWithLogger(
           this.logger_, 'Unexpected NaclModule type: ' + this.type);
       return '';
   }
-};
+}
 
 /**
  * @param {string} type
  * @param {function(?)} listener
  * @private
  */
-NaclModule.prototype.addEventListener_ = function(type, listener) {
+addEventListener_(type, listener) {
   this.element_.addEventListener(type, listener, true);
-};
+}
 
 /** @private */
-NaclModule.prototype.addStatusEventListeners_ = function() {
+addStatusEventListeners_() {
   this.addEventListener_('load', this.loadEventListener_.bind(this));
   this.addEventListener_('abort', this.abortEventListener_.bind(this));
   this.addEventListener_('error', this.errorEventListener_.bind(this));
   this.addEventListener_('crash', this.crashEventListener_.bind(this));
-};
+}
 
 /** @private */
-NaclModule.prototype.loadEventListener_ = function() {
+loadEventListener_() {
   if (this.isDisposed())
     return;
   goog.log.info(this.logger_, 'Successfully loaded NaCl module');
   this.loadPromiseResolver_.resolve();
-};
+}
 
 /** @private */
-NaclModule.prototype.abortEventListener_ = function() {
+abortEventListener_() {
   if (this.isDisposed())
     return;
   goog.log.error(
       this.logger_,
       'NaCl module load was aborted with the following ' +
           'message: ' + this.element_['lastError']);
   this.dispose();
-};
+}
 
 /** @private */
-NaclModule.prototype.errorEventListener_ = function() {
+errorEventListener_() {
   if (this.isDisposed())
     return;
   goog.log.error(
       this.logger_,
       'Failed to load NaCl module with the following ' +
           'message: ' + this.element_['lastError']);
   this.dispose();
-};
+}
 
 /** @private */
-NaclModule.prototype.crashEventListener_ = function() {
+crashEventListener_() {
   if (this.isDisposed())
     return;
   goog.log.error(this.logger_, 'The NaCl module has crashed');
   this.dispose();
-};
+}
 
 /** @private */
-NaclModule.prototype.forceElementLoading_ = function() {
+forceElementLoading_() {
   // Request the offsetTop property to force a relayout. As of June 29, 2014,
   // this is needed if the module is being loaded in a background page (see
   // crbug.com/350445).
   // Assign the result to a random property, so that Closure Compiler doesn't
   // optimize the "useless" expression away.
   this.element_.style.top = this.element_.offsetTop;
+}
 };
```

Comment:
Here too a comment on which class is being closed might be helpful

Reply:
Ack - see above.